zebrad/src/components/inbound/downloads.rs

```diff
@@ -49,7 +49,7 @@ type BoxError = Box<dyn std::error::Error + Send + Sync + 'static>;
 /// Since Zebra keeps an `inv` index, inbound downloads for malicious blocks
 /// will be directed to the malicious node that originally gossiped the hash.
 /// Therefore, this attack can be carried out by a single malicious node.
-pub const MAX_INBOUND_CONCURRENCY: usize = 20;
+pub const MAX_INBOUND_CONCURRENCY: usize = 30;
```

Comment:
Tangent: It would nice to make target memory usage configurable to scale some of these constants.

Reply:
Counterpoint: If these constants have reliability impacts, then we might not want users changing them. Every configuration is something extra we have to support.